subsys/nfc/lib/platform.c

```diff
@@ -11,6 +11,13 @@
 
 LOG_MODULE_REGISTER(nfc_platform, CONFIG_NFC_PLATFORM_LOG_LEVEL);
 
+#define NFC_TIMER_IRQn		NRFX_CONCAT_3(TIMER,				  \
```

Comment:
GH diff distorts code alignment. Here you can see how the macros look in the raw file:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/9f88a1baa5ac1a5df5045d6f7f690ce09bc4d726/subsys/nfc/lib/platform.c